csv-collector/cmd/fetch_csv.go

```diff
@@ -0,0 +1,116 @@
+// Copyright (c) 2020, Marcelo Jorge Vieira
+// Licensed under the AGPL-3.0+ License
+
+package cmd
+
+import (
+	"archive/zip"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"os"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+const (
+	firstYear = 2014
+	lastYear  = 2021 // TODO pegar de forma dinâmica o último ano suportado
```

Comment:
the is year is `2020`

Reply:
Maybe we can crawling the site bellow to get the correct "last year":
https://www.tse.jus.br/eleicoes/estatisticas/repositorio-de-dados-eleitorais-1/repositorio-de-dados-eleitorais

Iframe=> https://www.tse.jus.br/hotsites/pesquisas-eleitorais/candidatos.html

Reply:
Crawling can be fragile if the page is changed. Another possibility is the use of `time.Now().Year()` for lastYear and check status code to verify the zip file availability. What do you think?

Reply:
ok, go ahead! use `time.Now().Year()`

Reply:
Done!